src/constants/languages.js

```diff
@@ -109,24 +105,24 @@ const languages = [
   { value: 'tt', label: 'Tatar' },
   { value: 'te', label: 'Telugu' },
   { value: 'th', label: 'Thai' },
-  { value: 'bo-cn', label: 'Tibetan (PRC)' },
+  { value: 'bo', label: 'Tibetan' },
   { value: 've', label: 'Tshivenḓa' },
   { value: 've', label: 'Venḓa' },
```

Comment:
Similar to my comments on the Tsonga/Xitsonga labels, there are two possible labels for the Venda language. PFE as of the matching PR [7134](https://github.com/zooniverse/Panoptes-Front-End/pull/7134/files), only uses the "Xitsonga" label.

https://en.wikipedia.org/wiki/Venda_language

Again, no changes for this PR or the PFE one, but worth a follow up discussion.

Reply:
Same logic described [here](https://github.com/zooniverse/pandora/pull/848#discussion_r1657868216) applies in this case.